src/ontology/pato-edit.obo

```diff
@@ -22043,6 +22043,49 @@ is_a: PATO:0001655 ! osmolarity
 created_by: http://orcid.org/0000-0002-3410-4655
 creation_date: 2020-10-30T08:08:48Z
 
+[Term]
+id: http://purl.obolibrary.org/obo/PATO0070027
+name: intratelencephalic projection
+def: "A prolongation or process extending from a cell derived from the telencephalon to structures derived from the telencephalon." []
```

Comment:
A neuron projection quality that is a characteristic of neurons with both soma and projections entirely within the telencephalon.